boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -0,0 +1,6 @@
+#include "states/App_AllStates.h"
+
+void App_AllStatesRunOnTick1Hz(struct StateMachine *const state_machine)
+{
+    (void)state_machine;
```

Comment:
Adding some skeleton function to be filled in by the RGB LED sequence later.

Reply:
OK, let's use the `UNUSED` macro here just to be explicit and we know to remove it when we add actual code?

Reply:
Fixed